src/disk_interface.h

```diff
@@ -84,7 +84,7 @@ struct RealDiskInterface : public DiskInterface {
   /// Whether stat information can be cached.  Only has an effect on Windows.
   void AllowStatCache(bool allow);
 
-  /// Removes any cached stat information.  Only has an effect on Windows.
+  /// Removes any cached stat information. Only defined on Windows.
```

Comment:
Also wrap this declaration please - having a linker error when something tries to use it, would be weird. You can drop the note from the comment then.

Reply:
Done.